great_expectations/dataset/base.py

```diff
@@ -1032,23 +1032,19 @@ def _format_column_map_output(self,
             return return_obj
 
         # Try to return the most common values, if possible.
-        # If we have a dict, we probably had a dataframe; punt
-        if isinstance(unexpected_list, list):
-            try:
-                partial_unexpected_counts = [
-                    {'value': key, 'count': value}
-                    for key, value
-                    in sorted(
-                        Counter(unexpected_list).most_common(
-                            result_format['partial_unexpected_count']),
-                        key=lambda x: (-x[1], x[0]))
-                ]
-            except TypeError:
-                partial_unexpected_counts = [
-                    'partial_exception_counts requires a hashable type']
-        else:
-            partial_unexpected_counts = ['partial_unexpected_counts requires a flattened type']
-
+        try:
```

Comment:
I didn't follow the reason for the original change. Looks like we're back to the previous version?